src/Physics.jl

```diff
@@ -148,29 +148,29 @@ for name in [:mass, :angularmass, :position, :angle, :velocity, :angularvelocity
     end
 
     @eval function $(Symbol("set$(name)!"))(obj::Physical, val)
-        isnan(val) && error("value contains NaN: $val")
+        any(isnan.(val)) && error("value contains NaN: $val")
```

Comment:
Should probably be `any(isnan, val)` to allow specialisation on `typeof(isnan)`, though not a performance critical part, as it will error anyway. :woman_shrugging: